subprojects/gstreamer-vaapi/gst-libs/gst/vaapi/gstvaapidecoder_av1.c

```diff
@@ -712,6 +712,7 @@ av1_fill_picture_frame_header (GstVaapiDecoderAV1 * decoder,
   pic_param->pic_info_fields.bits.uniform_tile_spacing_flag =
       frame_header->tile_info.uniform_tile_spacing_flag;
   COPY_PIC_FIELD (allow_warped_motion);
+  COPY_PIC_FIELD (large_scale_tile);
```

Comment:
Please add a check for va-api version at compile time.   large_scale_tile is not available in old versions of libva.

Reply:
Please ignore the above comment.